src/api/api.ts

```diff
@@ -39,9 +33,31 @@ export const getTotalPosts = async (params: string): Promise<writing[]> => {
       console.error('Response data:', err.response.data);
       throw err;
     }
-
     console.error(error);
     throw error;
   }
 };
+export const tempSignUP = async () => {
+  const TEMP_URL = '/auth/signIn';
+  try {
+    if (!Cookies.get('accessToken')) {
+      const response = await instance.post(TEMP_URL, {
+        email: 'leedong0225@icloud.com',
+        password: 'abcd1234',
+      });
+      const { accessToken, refreshToken } = response.data;
+      Cookies.set('accessToken', accessToken);
+      Cookies.set('refreshToken', refreshToken);
+    }
+  } catch (error) {}
+};
+
+export const postImage = async () => {
+  const URL = '/images/upload';
+
+  try {
+    const response = await instance.post(URL);
+    response.data();
+  } catch (error) {}
+};
```

Comment:
만약 repsonse로 받은 값을 쓰지 않는다면 이렇게 해줘도 되겠네요!
```suggestion
export const postImage = async () => {
  const URL = '/images/upload';

  try {
     await instance.post(URL);
  } catch (error) {}
};
```

Reply:
감사합니다!